app/views/templates/report-details.html.slim

```diff
@@ -86,7 +86,8 @@
       .panel id='{{ testAnchor(result) }}' ng-class='{"panel-success": result.passed && result.active, "panel-danger": !result.passed && result.active, "panel-warning": !result.active}'
         .panel-body
           test-key-label.pull-right ng-if='result.key' key='result.key'
-          h4 ng-class='{"text-success": result.passed && result.active, "text-danger": !result.passed && result.active, "text-warning": !result.active}' {{ result.name }}
+          h4 ng-class='{"text-success": result.passed && result.active, "text-danger": !result.passed && result.active, "text-warning": !result.active}'
+            a href='#' ui-sref='org.tests.show({ testId: result.testId, orgId: result.project.organizationId })' {{ result.name }}
```

Comment:
In fact, I was confused by the GET /...?organizationId where the orgId is required. But without this query param, there is no need to the parameter in the state anymore.

Finally, `orgId` or `orgName` is not useful if we use `currentOrganization` to retrieve the organization name. The name is used to show <organization> > <project> under the test name which is the title of the page.

If it is ok for you, we can go with only test id:
```
a href='#' ui-sref='org.tests.show({ testId: result.testId })' {{ result.name }}
```
which result in simplification of the `TestCtrl`:
```
  .controller('TestCtrl', function (api, forms, orgs, projects, $scope, $state, $stateParams) {
    orgs.forwardData($scope);

    api({
      url: '/tests/' + $stateParams.testId,
      params: {
        withProject: 1,
        withContributions: 1
      }
    }).then(function(response) {
      $scope.test = response.data;
    });
  })
```
Before, there was a REST call to retrieve the organization data.

And the template is also updated to show the content only when the test has been loaded:
```
    h4 ng-if='!test' Loading...
    h4 ng-if='test' {{ currentOrganization | orgName }} > {{ test.project | projectName }}
```